aws_pricing/setup.py

```diff
@@ -45,15 +45,15 @@ def parse_pyproject_array(name):
 setup(
```

Comment:
We can drop the setup.py file if we're dropping py2 support; that way we avoid having to keep this in sync with pyproject.toml.